examples/asr/notebooks/7_VAD_Offline_Online_Microphone_Demo.ipynb

```diff
@@ -0,0 +1,642 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates voice activity detection from a microphone's stream (online) and a given wav file (offline)  in NeMo.\n",
+    "\n",
+    "It is **not a recommended** way to do inference in production workflows. If you are interested in \n",
+    "production-level inference using NeMo ASR models, please sign-up to Jarvis early access program: https://developer.nvidia.com/nvidia-jarvis"
```

Comment:
Please remove the section that talks about Jarvis. Since this isn't an `ASR` model.